apps/src/redux/instructions.js

```diff
@@ -4,73 +4,63 @@
  * that are required for this feature, and the reducer that sets state based
  * off of those actions.
  */
-var _ = require('../lodash');
+const TOGGLE_INSTRUCTIONS_COLLAPSED = 'instructions/TOGGLE_INSTRUCTIONS_COLLAPSED';
+const SET_INSTRUCTIONS_HEIGHT = 'instructions/SET_INSTRUCTIONS_HEIGHT';
+const SET_INSTRUCTIONS_MAX_HEIGHT = 'instructions/SET_INSTRUCTIONS_MAX_HEIGHT';
 
-var TOGGLE_INSTRUCTIONS_COLLAPSED = 'instructions/TOGGLE_INSTRUCTIONS_COLLAPSED';
-var SET_INSTRUCTIONS_HEIGHT = 'instructions/SET_INSTRUCTIONS_HEIGHT';
-var SET_INSTRUCTIONS_MAX_HEIGHT = 'instructions/SET_INSTRUCTIONS_MAX_HEIGHT';
-
-var instructionsInitialState = {
+const instructionsInitialState = {
   collapsed: false,
   // represents the uncollapsed height
   height: 300,
   maxHeight: 0
 };
 
-module.exports.default = function reducer(state, action) {
-  state = state || instructionsInitialState;
-
+export default function reducer(state = instructionsInitialState, action) {
   if (action.type === TOGGLE_INSTRUCTIONS_COLLAPSED) {
-    return _.assign({}, state, {
+    return Object.assign({}, state, {
       collapsed: !state.collapsed
     });
   }
 
   if (action.type === SET_INSTRUCTIONS_HEIGHT &&
       action.height !== state.height) {
-    return _.assign({}, state, {
+    return Object.assign({}, state, {
       height: action.height
     });
   }
 
   if (action.type === SET_INSTRUCTIONS_MAX_HEIGHT &&
       action.maxHeight !== state.maxHeight) {
-    return _.assign({}, state, {
+    return Object.assign({}, state, {
       maxHeight: action.maxHeight
     });
   }
 
   return state;
-};
+}
 
 /**
  * Toggles whether instructions are currently collapsed.
  */
-module.exports.toggleInstructionsCollapsed = function () {
-  return {
-    type: TOGGLE_INSTRUCTIONS_COLLAPSED
-  };
-};
+export const toggleInstructionsCollapsed = () => ({
+  type: TOGGLE_INSTRUCTIONS_COLLAPSED
+});
 
 /**
  * Set the height of the instructions panel
  * @param {number} height - Height of instructions pane
  */
-module.exports.setInstructionsHeight = function (height) {
-  return {
-    type: SET_INSTRUCTIONS_HEIGHT,
-    height: height
-  };
-};
+export const setInstructionsHeight = height => ({
+  type: SET_INSTRUCTIONS_HEIGHT,
+  height
+});
 
 /**
  * Set the max height of the instructions panel
  * @param {number} maxHeight - Don't let user drag instructions pane to be
  *   larger than this number.
  */
-module.exports.setInstructionsMaxHeight = function (maxHeight) {
-  return {
-    type: SET_INSTRUCTIONS_MAX_HEIGHT,
-    maxHeight: maxHeight
-  };
-};
+export const setInstructionsMaxHeight = maxHeight => ({
```

Comment:
I don't have a strong preference here. I like getting to use arrows everywhere that I can, but I'm not sure I have a great justification for why that is.

One thing to note about the example you reference in 22.6, where they do:
`export default function fortyTwo() { return 42; }`
This can't actually be done as a one liner with a const. The following is not valid:
`export default const fortyTwo = () => 42`
You instead would need to do
```
const fortyTwo = () => 42;
export default const fortyTwo
```

Might be another argument for preferring top level functions use the function keyword.